.clang-tidy-diff-scans.txt

```diff
@@ -19,6 +19,9 @@
 
 ./iceoryx_hoofs/include/iceoryx_hoofs/posix_wrapper/*
 ./iceoryx_hoofs/include/iceoryx_hoofs/internal/posix_wrapper/*
+
+./iceoryx_hoofs/include/iceoryx_hoofs/design_pattern/builder.hpp
```

Comment:
It seems clang-tidy has some issues with `creation.hpp`. Since this will be removed anyway, only the `builder.hpp` is checked.